emcc.py

```diff
@@ -2435,6 +2442,11 @@ def consume_arg():
       options.extern_pre_js += open(consume_arg()).read() + '\n'
     elif check_arg('--extern-post-js'):
       options.extern_post_js += open(consume_arg()).read() + '\n'
+    elif check_arg('--oformat'):
+      options.oformat = consume_arg()
+      formats = ('js', 'html', 'wasm', 'mjs')
```

Comment:
this could also be a CONSTANT defined early in the file? again, i don't feel strongly.

Reply:
For single use stuff like this I prefer locality of reference I think.